build.sbt

```diff
@@ -21,4 +21,6 @@ lazy val intellijHaskell = (project in file(".")).
     unmanagedJars in Compile += baseDirectory.value / "idea"
   )
 
-ideaBuild in ThisBuild := "163.7743.44"
\ No newline at end of file
+ideaBuild in ThisBuild := "163.7743.44"
+
+javacOptions ++= Seq("-encoding", "UTF-8")
```

Comment:
My default encoding is UTF-8 in IntelliJ. I can not reproduce your issue. Does your change in build.sbt works for you?